app/components/primer/navigation/tab_component.rb

```diff
@@ -18,10 +18,22 @@ class TabComponent < Primer::Component
       #
       # @param system_arguments [Hash] <%= link_to_system_arguments_docs %>
       renders_one :panel, lambda { |**system_arguments|
+        return unless @with_panel
+
         system_arguments[:tag] = :div
         system_arguments[:role] ||= :tabpanel
+        system_arguments[:tabindex] = 0
         system_arguments[:hidden] = true unless @selected
 
+        label_present = system_arguments[:"aria-label"] || system_arguments.dig(:aria, :label) || system_arguments[:"aria-labelledby"] || system_arguments.dig(:aria, :labelledby)
```

Comment:
I think it is time to write helpers to get aria values, this double check is getting propagated everywhere and it's not great 😅 

We could do something like this in `Primer::Component`:
```rb
def aria(val, system_arguments)
  system_arguments[:"aria-#{val}"] || system_arguments.dig(:aria, val)
end
```

Reply:
great idea!

Reply:
I may add a linter in a followup.